src/main/java/FileOperator.java

```diff
@@ -0,0 +1,40 @@
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileOperator {
+    private static int rows;
+    private static int columns;
+
+    public static int[][] readBoard(Path p) throws IOException {
+        if (Files.exists(p)) {
+            List<String> lines = Files.readAllLines(p);
+            int[][] board = new int[lines.size()][lines.size() + 1];
+            int i = 0;
+            int j = 0;
+            for (String s : lines) {
+                for (String arg : s.split("\\s+")) {
+                    board[i][j] = Integer.parseInt(arg);
+                    j++;
+                }
+                j = 0;
+                i++;
+            }
```

Comment:
if you need `i` and `j`, it should be better to use clasic for (`for(int i = 0; i<lines.length; i++)`).